torchaudio/functional.py

```diff
@@ -0,0 +1,358 @@
+import numpy as np
+import torch
+
+
+__all__ = [
+    'scale',
+    'pad_trim',
+    'downmix_mono',
+    'LC2CL',
+    'spectrogram',
+    'create_fb_matrix',
+    'mel_scale',
+    'spectrogram_to_DB',
+    'create_dct',
+    'MFCC',
+    'BLC2CBL',
+    'mu_law_encoding',
+    'mu_law_expanding'
+]
+
+
+def scale(tensor, factor):
+    # type: (Tensor, int) -> Tensor
+    """Scale audio tensor from a 16-bit integer (represented as a FloatTensor)
```

Comment:
This documentation isn't quite right, because this function literally just converts to float32 and divides the input. Let's update this in the next PR.